site/guide/model-inventory/register-models-in-inventory.qmd

```diff
@@ -17,19 +17,28 @@ To be able to document models using ValidMind's model documentation and validati
 2. Click **Model Inventory** in the left sidebar. 
 <!--- **Need screenshot** --->
    
-3. Click **Register new model**.
+3. Click **Register New Model**.
    
-4. Fill in the required information on the registration form:
+4. Depending on whether or not your organization has custom fields[^2] defined, input the information on the Register New Model panel in either one or two steps:
 
-   - Provide a model name
-   - Select the relevant business unit
-   - Select the relevant model methodology being used
-   - Select the relevant model use case
-   - Provide a purpose statement to explain what the model will be used for
-   - Select the preliminary risk tier for the model
-   - If the model is a vendor model, toggle **Is Vendor Model**[^2] and provide the vendor name.
+   - **Step 1: Basic Model Information** — If no custom fields are defined for your models, simply provide the following information then move on to step #5 below: 
 
-5. Click **Register new model** to create a new entry in the model inventory.
+      - Model name
+      - Type of model, new[^3] or existing
+      - Associated business unit
+      - Use case
+      - If the model is a vendor model, toggle **Is Vendor Model**[^4] and provide the vendor name
+      - Purpose: Explain what the model will be used for
+      - Preliminary risk tier 
+
+   - **Step 2: Additional Model Information** — If custom fields are defined for your organization:
```

Comment:
Custom fields were defined for my organization, however this step didn't apply. 
HOWEVER I created a "Required on registration" custom field and then this step applied.

Reply:
Hm, are you saying that the step 2 panel didn't show up at all even though you had custom fields on your organization?